docs/source/getting-started/tabular-data/computing-basic-statistics.ipynb

```diff
@@ -0,0 +1,472 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Computing Basic Statistics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this section, you will learn to compute essential statistical measures such as: \n",
+    "\n",
+    "* Sum \n",
+    "* Mean \n",
+    "* Median\n",
+    "* Quantiles\n",
+    "\n",
+    "For each method, we will compare the actual values to the differentially private values to demonstrate utility. The [documentation](../../api/python/opendp.polars.html) also provides more information about the methods. We will use the [sample data](https://github.com/opendp/dp-test-datasets/blob/master/data/sample_FR_LFS.csv.zip) from the Labour Force Survey in France. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set Up"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import polars as pl \n",
+    "import opendp.prelude as dp\n",
+    "\n",
+    "dp.enable_features(\"contrib\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+      "  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\n",
+      "  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\n",
+      "100 5933k  100 5933k    0     0  1022k      0  0:00:05  0:00:05 --:--:-- 1100k\n",
+      "Archive:  data.zip\n",
+      "  inflating: sample_FR_LFS.csv       \n",
+      "  inflating: __MACOSX/._sample_FR_LFS.csv  \n"
+     ]
+    }
+   ],
+   "source": [
+    "![ -e sample_FR_LFS.csv ] || ( curl 'https://github.com/opendp/dp-test-datasets/blob/master/data/sample_FR_LFS.csv.zip?raw=true' --location --output sample_FR_LFS.csv.zip; unzip sample_FR_LFS.csv.zip )\n",
+    "\n",
+    "df = pl.scan_csv(\"sample_FR_LFS.csv\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The compositor here is nearly identical to the one explained in the introduction. One additional parameter that is included is the `max_num_partitions.` This is required when the metric isn't sensitive to ordering. In the following examples, there is only one partition in select, so `max_num_partitions` is set to 1. "
```

Comment:
- I would put a link to the "introduction" when we say that word.
- I would also specify what this max_num_partitions is doing in a sentence or so.